src/components/Proposal/Create/CreateProposal.scss

```diff
@@ -1,6 +1,10 @@
 @import "./react-mde.css";
 :global {
   @import "../../../../node_modules/react-datetime/css/react-datetime";
+
+  body.react-router-modal__modal-open {
```

Comment:
Does this not also need to be in ModalPopup.scss, to cover all the other possible popups?

Reply:
Agree, but first I want to locate those popup and see that it's affecting.

Reply:
Okay looks like the above code also affecting ModalPopup.